Document rot_wrt_axis parameter

cdeil opened the pull request:
This is a minor cleanup to make `rot_wrt_axis` default keyword arguments bool instead of int, to make it clearer that this is a flag (and not e.g. a float rotation angle).

@leejjoon - OK?

leejjoon commented:
'rot_wrt_axis' is from the PR (#34) by @mcara. And I guess he is using this with value of 2. So, this may affect his codes. And the name of the keyword means (I guess) "rotation with respect to axis" 1 or 2, so I guess 1 or 2 makes more sense that True/False.  So, unless @mcara is supportive of this PR, I would rather leave it as is, but add an adequate documentation.

cdeil commented:
@leejjoon @mcara - Thanks for the infos.

I've changed the commit here to document `rot_wrt_axis` as you explained:
```
        rot_wrt_axis : {1, 2}
            Use rotation with respect to axis 1 (X-axis) or axis 2 (Y-axis) and north.
```
I've also changed the use to raise a `ValueError` if the user passes something other than 1 or 2.
